samples-ts/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "samples-ts",
+    "version": "1.0.0",
+    "description": "",
+    "main": "dist/functions/*.js",
+    "scripts": {
+        "build": "tsc",
+        "watch": "tsc -w",
+        "prestart": "npm run build",
+        "start": "func start",
+        "test": "echo \"No tests yet...\""
+    },
+    "devDependencies": {
+        "@azure/functions": "file:../../../nodejs-worker/library",
```

Comment:
This should reference an actual version. And should probably be in dependencies (I know it's technically already a sub dependency of durable-functions, but still)

Reply:
Good call out, thanks! Done

Reply:
@hossam-nasr you fixed the version, but it should still be moved from `devDependencies` to `dependencies`

Reply:
Done, thanks